modules/python-modules/syslogng/modules/kubernetes/scl/kubernetes.conf

```
@@ -115,7 +116,7 @@ block parser kubernetes-file-parser(prefix('.k8s.') cluster-name('k8s')) {
         timeout(10)
     );
 
-    kubernetes-metadata-parser(prefix(`prefix`));
+    kubernetes-metadata-parser(prefix(`prefix`) key-delimiter(`key-delimiter`);
```

A closing bracket is missing.
```suggestion
    kubernetes-metadata-parser(prefix(`prefix`) key-delimiter(`key-delimiter`));
```